libs/auth-api-lib/src/lib/delegations/names.service.ts

```diff
@@ -32,12 +32,14 @@ export class NamesService {
   }
 
   async getPersonName(nationalId: string) {
-    const person = await this.nationalRegistryClient.getIndividual(nationalId)
+    const person = await this.nationalRegistryClient.getAllDataIndividual(
+      nationalId,
+    )
     if (!person) {
       throw new BadRequestException(
         `A person with nationalId<${nationalId}> could not be found`,
       )
     }
-    return person.fullName ?? person.name
+    return person.fulltNafn?.fulltNafn ?? person.nafn ?? ''
```

Comment:
We may want to be using person.nafn rather than person.fulltNafn. Was contacted by Þórður at Hugsmiðjan about fulltNafn not returning updated value for a name that had changed.